.github/workflows/rebuild-latest-docs.yml

```diff
@@ -0,0 +1,39 @@
+name: Rebuild latest docs
+
+# PROCESS
+#
+# 1. Build User Guide and API docs
+# 2. Publish to GitHub Pages
```

Comment:
I don't think we need this?

Reply:
it just describes steps of the process which is done here (except "and API docs" which can be removed). I think it might be helpful to understand what we do here.

Reply:
The step "# 2. Publish to GitHub Pages" - we don't need this anymore, do we - maybe it becomes "push versioned docs to gh-pages branch" or something?